### PR TITLE
[RFR] Added ssh credentials handling

### PR DESCRIPTION
**Pull request by grafuls**

### CODE REVIEW

File: disruption_generator/parsers/experiment_parser.py [hunk 1]
```diff
@@ -89,15 +115,23 @@ def _init_actions(element_trigger):
 
         _scenarios = []
         for disrupt_action in doc:
-            name = disrupt_action[config.DISRUPT_ACTION_STR][0][config.NAME_STR]
+            name = disrupt_action[config.DISRUPT_ACTION_KEY][0][
```
Comment (jan-zmeskal) [inline disruption_generator/parsers/experiment_parser.py]: Why is there [0] index? In example.yaml, `name` key is directly under `disrupt_action`.

File: disruption_generator/trigger/trigger.py [hunk 2]
```diff
@@ -1,24 +1,38 @@
...
+        self.username = username
+        self.password = password
+        self.ssh_host_key = ssh_host_key
 
     def __getattr__(self, item):
-        attr_err_msg = "Unexpected disruptive action other than %s" % ", ".join(
-            ALL_ACTIONS
+        attr_err_msg = (
+            "Unexpected disruptive action other than %s"
+            % ", ".join(ALL_ACTIONS)
```
Comment (jan-zmeskal) [inline disruption_generator/trigger/trigger.py]: Please use new-style string formatting.

File: disruption_generator/parsers/config.py [hunk 3]
```diff
@@ -4,8 +4,18 @@
 Helper config
 """
 # parsers
-DISRUPT_ACTION_STR = "disrupt_action"
-NAME_STR = "name"
-LISTENER_STR = "listener"
-TRIGGER_STR = "trigger"
+ACTION_KEY = "action"
```
[the remark this answers is not in the document]
Comment (jan-zmeskal) [reply disruption_generator/parsers/config.py]: This is what parses should be looking for in experiment definition file, e.g. this one: https://github.com/grafuls/disruption_generator/blob/master/disruption_generator/experiments/example.yaml

File: disruption_generator/parsers/experiment_parser.py (hunk 1 above)
Comment (grafuls) [reply disruption_generator/parsers/experiment_parser.py]: Cause it bring a 1 element list

**Comment by grafuls** [comment]
@jan-zmeskal @StLuke Please review additional changes based on your comments.

**Pull request merged**